pkg/apis/core/v1alpha1/types_project.go

```diff
@@ -89,13 +89,16 @@ type ProjectStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"varint,1,opt,name=observedGeneration"`
 	// Phase is the current phase of the project.
 	Phase ProjectPhase `json:"phase,omitempty" protobuf:"bytes,2,opt,name=phase,casttype=ProjectPhase"`
+	// LastActivityTimestamp contains the timestamp from the last activity performed in this project.
+	// +optional
+	LastActivityTimestamp *metav1.Time `json:"lastActivityTimestamp,omitempty" protobuf:"bytes,3,opt,name=lastActivityTimestamp"`
```

Comment:
Changing the order in protobuf is an incompatible change. Thus, you need to append the new field.